Springsource now forces HTTPS

Not specifying HTTPS on this line breaks the maven build of WebAPI.

[ERROR] Failed to execute goal on project WebAPI: Could not resolve dependencies for project org.ohdsi:WebAPI:war:2.7.5: Failed to collect dependencies at org.springframework.batch:spring-batch-admin-manager:jar:2.0.0.M1: Failed to read artifact descriptor for org.springframework.batch:spring-batch-admin-manager:jar:2.0.0.M1: Could not transfer artifact org.springframework.batch:spring-batch-admin-manager:pom:2.0.0.M1 from/to springsource.releases (http://repo.springsource.org/release): Access denied to: http://repo.springsource.org/release/org/springframework/batch/spring-batch-admin-manager/2.0.0.M1/spring-batch-admin-manager-2.0.0.M1.pom , ReasonPhrase:Forbidden.

https://github.com/OHDSI/WebAPI/blob/f0c549233df5271fda0e0b0242b579b9bccba899/pom.xml#L460

--- a/pom.xml
+++ b/pom.xml
@@ -457,7 +457,7 @@
     <repository>
       <id>springsource.releases</id>
       <name>SpringSource GA Repository</name>
-      <url>http://repo.springsource.org/release</url>
+      <url>https://repo.springsource.org/release</url>
       <releases>
         <enabled>true</enabled>
       </releases>

